Update code resend button cooldown based on rate limit

carmenlau opened the pull request:
fix #953 

~~WIP: After changing the button state based on rate limit, the resend button can be clicked multiple times without cooldown. So now the resend link will not change to count down after click. We need to add some visual feedback to let the user know the code is resent.~~

Updated using flash message to show "resend code successfully message".

louischan-oursky commented on authui/src/authgear.ts:
```diff
@@ -258,6 +258,38 @@ window.api.onLoad(() => {
+// When clicking element with `data-submit-link`, it will perform click on
+// element with `data-submit-form` that contains the same value
+// e.g. data-submit-link="verify-identity-resend" and
+//      data-submit-form="verify-identity-resend"
```
This is smart!

louischan-oursky reviewed (commented):
LGTM! Just some very minor suggestions

carmenlau commented:
Updated

carmenlau commented:
Added 2 commits to address the problems we mentioned in the product meeting.
1. Added new rate limit keys listed below, finally i didn't put channel (email/sms) into the key since it is not useful for counting this rate limit.
    - `verification-send-code:{target}`
    - `oob-send-code:setup-primary-oob:{target}`
    - `oob-send-code:setup-secondary-oob:{target}`
    - `oob-send-code:authenticate-primary-oob:{target}`
    - `oob-send-code:authenticate-secondary-oob:{target}`
2. Enforce verification code_expiry_seconds > 60
